core/src/Ribasim.jl

```diff
@@ -13,12 +13,15 @@ using DataInterpolations: LinearInterpolation, derivative
 using Dates
 using DBInterface: execute, prepare
 using Dictionaries: Indices, Dictionary, gettoken, gettokenvalue, dictionary
+using ForwardDiff: pickchunksize
 using DiffEqCallbacks
 using Graphs: DiGraph, add_edge!, adjacency_matrix, inneighbors, outneighbors
 using Legolas: Legolas, @schema, @version, validate, SchemaVersion, declared
 using Logging: current_logger, min_enabled_level, with_logger
 using LoggingExtras: EarlyFilteredLogger, LevelOverrideLogger
 using OrdinaryDiffEq
+using PreallocationTools: DiffCache, get_tmp
+using Preferences: set_preferences!
```

Comment:
You no longer use this right? Then it can be removed from here and the project.

Also, since the dependencies changed, could you `pkg> resolve` the `libribasim` and `ribasim_cli` manifests to reflect this change?

Reply:
Did I do this correctly?